data/lm/generate_lm.py

```diff
@@ -103,7 +126,7 @@ def build_lm(args, data_lower, vocab_str):
         [
             os.path.join(args.kenlm_bins, "filter"),
             "single",
-            "model:{}".format(lm_path),
+            f"model:{lm_path}",
```

Comment:
This might break on Python 3.5 I think

Reply:
Happy to revert these `.format` changes if you recommend. I was just thinking it's easier to read plus 3.5 now seems to be EOL:
https://www.python.org/downloads/release/python-3510/

Reply:
It is, but for now we still have CI running on it. This will change when we move to TensorFlow r2.4, but for now I feel it's an easy change.